README.md

```diff
@@ -1,9 +1,12 @@
 # TypeScript and JavaScript lib generator
 
-Travis CI Status: [![Build Status](https://travis-ci.org/Microsoft/TSJS-lib-generator.svg?branch=master)](https://travis-ci.org/Microsoft/TSJS-lib-generator)
+This tool is used to generate the web-based `*.d.ts` files which are included with TypeScript releases, and as `@types/x` packages.
```

Comment:
A little more concrete, a little less complete:

```suggestion
This tool is used to generate the web-based `lib.dom.d.ts` file which is included with TypeScript releases, and as the `@types/web` package.
```

We can expand this later when we're publishing more things from it. I don't think webworker and dom.iterable are worth calling out in the first line.

Reply:
Yah, they've not been added yet